recipes/folly/all/conanfile.py

```diff
@@ -244,29 +269,32 @@ def package_info(self):
         ]
         if not is_msvc(self):
             self.cpp_info.components["libfolly"].requires.append("libdwarf::libdwarf")
-        if self.settings.os == "Linux":
+        if self.settings.os in ["Linux", "FreeBSD"]:
             self.cpp_info.components["libfolly"].requires.extend(["libiberty::libiberty", "libunwind::libunwind"])
             self.cpp_info.components["libfolly"].system_libs.extend(["pthread", "dl", "rt"])
 
-        if Version(self.version) >= "2020.08.10.00":
-            self.cpp_info.components["libfolly"].requires.append("fmt::fmt")
-            if self.settings.os == "Linux":
-                self.cpp_info.components["libfolly"].defines.extend(["FOLLY_HAVE_ELF", "FOLLY_HAVE_DWARF"])
+        self.cpp_info.components["libfolly"].requires.append("fmt::fmt")
```

Comment:
Judging by this https://github.com/facebook/folly/blob/main/CMake/folly-deps.cmake#L307C1-L307C22 `fmt::fmt` may be as well the first "requires" of libfolly as it's the first element that getds added to folly_deps_INTERFACE_LINK_LIBRARIES